app/main/views/services.py

```diff
@@ -503,6 +503,62 @@ def update_section_submission(framework_slug, lot_slug, service_id, section_id,
                                 service_id=service_id))
 
 
+@main.route('/frameworks/<framework_slug>/submissions/<lot_slug>/<service_id>/remove/<question_slug>',
```

Comment:
Should we keep the URLs similar to section edits with `/remove/<section_id>/<question_slug>`? That way if we ever add a way to clear section contents too we won't have a URL clash.